habitat-lab/habitat/tasks/rearrange/utils.py

```diff
@@ -453,16 +453,36 @@ def place_agent_at_dist_from_pos(
             )
     else:
         return _get_robot_spawns(
-            target_position,
             rotation_perturbation_noise,
             distance_threshold,
             sim,
             num_spawn_attempts,
             filter_colliding_states,
+            target_position,
             agent=agent,
         )
 
 
+def random_place_agent(
+    sim,
+    num_spawn_attempts: int,
+    filter_colliding_states: bool,
+    agent: Optional[MobileManipulator] = None,
+):
+    """
+    Places the robot randomly
```

Comment:
Since this is function is publicly exposed, document the parameters and return types.

Reply:
Actually, you should use the existing `sim.set_articulated_agent_base_to_random_point` function instead and remove this.

Reply:
Done, revert back to the original design